src/faebryk/core/trait.py

```diff
@@ -110,6 +110,12 @@ def _handle_added_to_parent(self):
     def on_obj_set(self): ...
 
     def handle_duplicate(self, other: "TraitImpl", node: Node) -> bool:
+        """
```

Comment:
@IoannisP-ITENG I'm not sure this is 100% accurate. It'd be handy to know what the return means?

Reply:
FIxed in #113